dias/analyzers/sensitivity_analyzer.py

```diff
@@ -0,0 +1,273 @@
+from dias import CHIMEAnalyzer
+from datetime import datetime
+import calendar
+from caput import config
+from dias.utils.string_converter import str2timedelta, datetime2str
+from ch_util import data_index
+
+import os
+import sys
+import glob
+import time
+import subprocess
+import gc
+
+import h5py
+import numpy as np
+import scipy.constants
+
+from collections import Counter
+from collections import defaultdict
+
+from ch_util import andata
+from ch_util import tools
+from ch_util import ephemeris
+
+
+class SensitivityAnalyzer(CHIMEAnalyzer):
+    """SensitivityAnalyzer.
+    This subclass of dias.analyzer.Analyzer describes the analyzer.
```

Comment:
Can you remove this line that is copied from the sample analyzer and instead add a description of what your analyzer does?
Also please add a section that describes it's config variables, like that:
```
Attributes
--------------
sep_cyil : Bool
    Description....
```
(see https://github.com/chime-experiment/dias/blob/d3679591f2d6f14c49fc84ddf9f85b4484e6b573/dias/analyzers/feedpositions_analyzer.py#L41)
Note that you should only describe the config variables of `SensitivityAnalyzer` and not the ones of the analyzer base class and that `SensitivityAnalyzer` currently redefines some of the latter.

Can you include also what some of the default values mean or why they are chosen (nfreq_per_block, cyl_start_char, cyl_start_num)?

I am currently putting together a sphinx build that will make use of that. We also want to add a way to formally describe the data an analyzer writes. Until that is defined, can you describe the written data informally? 
I think it should cover descriptions of datasets, axis, other attributes and data types, but I don't know hdf5 much. Maybe @ketiltrout can add to this.